sys/include/net/gnrc/csma_sender.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2015 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_csma_sender  Helper inerface to send packets via CSMA/CA
+ * @ingroup     net_gnrc
+ * @brief       This interface allows code from layer 2 (MAC) or higher
+ *              to send packets with CSMA/CA, whatever the abilities and/or
+ *              coniguration of a iven radio transceiver device.
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the CSMA/CA helper
+ *
+ * @author      Kévin Roussel <Kevin.Roussel@inria.fr>
+ */
+
+#ifndef GNRC_CSMA_SENDER_H_
+#define GNRC_CSMA_SENDER_H_
+
+#include "kernel.h"
+#include "net/gnrc/netdev.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @brief Default Minimal CSMA/CA Backoff Exponent 
+ */
+#define MAC_MIN_BE_DEFAULT              (3U)
+
+/**
+ * @brief Default Maximal CSMA/CA Backoff Exponent 
+ */
+#define MAC_MAX_BE_DEFAULT              (5U)
+
+/**
+ * @brief Default Maximal number of retries for sending
+ *        a given packet with the CSMA/CA method
+ */
+#define MAC_MAX_CSMA_BACKOFFS_DEFAULT   (4U)
+
+/**
+ * @brief CSMA/CA backoff period, in microseconds
+ */
+#define A_UNIT_BACKOFF_PERIOD_uS      (320U)
+
+
+/**
+ * @brief Define a different (non-standard) value for
+ *        the CSMA macMinBE parameter.
+ *
+ * @param[in] val     new value for macMinBE
```

Comment:
Could you add a note to the top level documentation for this module just to let everyone know where to look up the meaning of the parameters. Add a reference to the standards document and a link to https://standards.ieee.org/about/get/802/802.15.html